js/controller.js

```diff
@@ -1091,6 +1091,7 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
         this.state.screenToShow = CONSTANTS.SCREEN.SHARE_SCREEN;
       } else {
         this.state.screenToShow = CONSTANTS.SCREEN.END_SCREEN;
+        this.renderSkin();
```

Comment:
Have you got any idea why `this.renderSkin()` shouldn't be moved into 1106 line? Seem like the screens from other conditions of this method are also would be happy to get rendered.